modules/manage/pages/audit-logging.adoc

```diff
@@ -1,53 +1,117 @@
 = Audit Logging
 :description: Learn how to use Redpanda's audit logging capabilities.
 
-Many scenarios for streaming data include the need for fine-grained auditing of user activity related to the system. This is especially true for regulated industries such as finance, healthcare, and the public sector. Complying with PCI DSS v4 standards, for example, requires verbose and detailed activity auditing, alerting, and analysis capabilities.
+Many scenarios for streaming data include the need for fine-grained auditing of user activity related to the system. This is especially true for regulated industries such as finance, healthcare, and the public sector. Complying with https://pcidssguide.com/whats-new-in-pci-dss-v4-0/[PCI DSS v4] standards, for example, requires verbose and detailed activity auditing, alerting, and analysis capabilities.
 
-Redpanda's auditing capabilities support recording both administrative and operational interactions with topics and with users. Redpanda complies with the https://schema.ocsf.io/[Open Cybersecurity Schema Framework (OCSF)], providing a predictable and extensible solution that works seamlessly with industry standard tools.
+Redpanda's auditing capabilities support recording both administrative and operational interactions with topics and with users. Redpanda complies with the Open Cybersecurity Schema Framework (OCSF), providing a predictable and extensible solution that works seamlessly with industry standard tools.
 
-NOTE: All audit logging is configured at the cluster level. Redpanda supports excluding specific topics or principals from audit to help reduce noise in the log. Audit logging is disabled by default.
+NOTE: Audit logging is configured at the cluster level. Redpanda supports excluding specific topics or principals from auditing to help reduce noise in the log. Audit logging is disabled by default.
 
 == Audit log flow
 
-The Redpanda audit log mechanism functions similar to the Kafka flow you may be familiar with. When a user interacts with another user or with a topic, Redpanda writes an event to a specialized audit topic. The audit topic is immutable. Only Redpanda itself can write to it -- users are prevented from writing to the topic directly and the Kafka API cannot create or delete it.
+The Redpanda audit log mechanism functions similar to the Kafka flow you may be familiar with. When a user interacts with another user or with a topics, Redpanda writes an event to a specialized audit topic. The audit topic is immutable. Only Redpanda itself can write to it -- users are prevented from writing to the topic directly and the Kafka API cannot create or delete it.
 
-image:shared:audit-logging-flow.png[Audit log flow]
+image:shared:audit-loging-flow.png[Audit log flow]
```

Comment:
Just curious why you changed the image title to use incorrect spelling?

Reply:
I think this was something that got dorked in trying to rebase. I fixed it and it looks like the fix got un-fixed. I'll re-fix it.